cassandra/cluster.py

```diff
@@ -3409,3 +3409,24 @@ def get_all_query_traces(self, max_wait_sec_per=None):
         See :meth:`.ResponseFuture.get_all_query_traces` for details.
         """
         return self.response_future.get_all_query_traces(max_wait_sec_per)
+
+    @property
+    def was_applied(self):
+        """
+        For LWT results, returns whether the transaction was applied.
+
+        Result is indeterminate if called on a result that was not an LWT request.
+
+        Only valid when one of tne of the internal row factories is in use.
+        """
+        if self.response_future.row_factory not in (named_tuple_factory, dict_factory, tuple_factory):
+            raise RuntimeError("Cannot determine LWT result with row factory %s" % (self.response_future.row_factsory,))
```

Comment:
core driver uses all '%', so this is consistent within that layer. cqlengine was using ``format()`` when we brought it in, but it had to be updated because it used unnamed placeholders. I haven't had time or cause to align the two so I've just been keeping it consistent within each package.